TMBuilder/src/main/java/com/ebay/tool/thinmodelgen/gui/menu/TMBuilderMenu.java

```diff
@@ -1073,7 +1089,7 @@ public String getDescription() {
 
   }
 
-  class JavaFileFilter extends FileFilter {
+  class AndroidFileFilter extends FileFilter {
```

Comment:
Why not use the AndroidFilenameFilter class instead and remove this inner class?

Reply:
This is something was existing so I left as is.